### PR TITLE
[PLAT-1528] Adding events for delta changed and interaction started

### PR DESCRIPTION
**Pull request by MadisonEhlers-Vertex**
## Summary

These changes introduce two new events. Events emitted when the widget delta position changes, as well as an event for when the widget has begin an interaction.

### CODE REVIEW

File: packages/viewer/src/components.d.ts [hunk 1]
```diff
@@ -2669,6 +2669,14 @@ declare namespace LocalJSX {
      * @ignore Visible for testing.
      */
     hovered?: Mesh;
+    /**
+     * An event that is emitted when the delta changed
+     */
+    onDeltaChanged?: (event: CustomEvent<Matrix4.Matrix4 | undefined>) => void;
+    /**
+     * An event that is emitted when the delta changed
```
Comment (MadisonEhlers-Vertex) [inline packages/viewer/src/components.d.ts:2677]: updating this to be correct

File: packages/viewer/src/components/viewer-transform-widget/viewer-transform-widget.tsx [hunk 2]
```diff
@@ -34,6 +34,18 @@ export class ViewerTransformWidget {
   @Event({ bubbles: true })
   public positionChanged!: EventEmitter<Vector3.Vector3 | undefined>;
 
+  /**
+   * An event that is emitted when the delta changed
+   */
+  @Event({ bubbles: true })
+  public deltaChanged!: EventEmitter<Matrix4.Matrix4 | undefined>;
```
Comment (jdm717) [inline packages/viewer/src/components/viewer-transform-widget/viewer-transform-widget.tsx:41]: How would you feel about calling this `interactionEnded` and providing a transform? Just to fit with the naming convention of the other event 🙂

File: packages/viewer/src/components/viewer-transform-widget/viewer-transform-widget.tsx [hunk 3]
```diff
@@ -34,6 +34,18 @@ export class ViewerTransformWidget {
...
+
+  /**
+   * An event that is emitted when the delta changed
+   */
```
Comment (jdm717) [inline packages/viewer/src/components/viewer-transform-widget/viewer-transform-widget.tsx:45]: nit: copied description

**Review by jdm717 — COMMENTED** [review]
LGTM aside from comments!

**Review by jdm717 — APPROVED** [review]

**Pull request merged**